tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -35,10 +37,12 @@ Verify Model Can Be Deployed Via UI
     [Tags]    Sanity    Tier1
     ...    ODS-1921
     Open Model Serving Home Page
-    # Verify No Models Are Present
-    Click Button    Create server
-    Wait Until Page Contains    Data science projects
-    Wait Until Page Contains Element    //button[.="Create data science project"]
+    Clean Up Model Serving Page
+    Clean Up DSP Page
+    Open Model Serving Home Page
+    Try Opening Create Server
+    SeleniumLibrary.Wait Until Page Contains    Data science projects
+    SeleniumLibrary.Wait Until Page Contains Element    //button[.="Create data science project"]
     ${already_present} =    Run Keyword And Return Status    Page Should Contain    ${PRJ_TITLE}
```

Comment:
probably not needed anymore because `Clean Up DSP Page` is called at the beginning of the test. Right?

Reply:
You're probably right, but this check shouldn't take more than a few ms